src/lib.ts

```diff
@@ -16,11 +16,13 @@ import {
 import {FontAwesomeIcon} from '@fortawesome/vue-fontawesome'
 //To register components globally by default
 import * as commonComponents from '@/components/common'
+import * as toolbarComponents from '@/components/interaction-designer/toolbar'
```

Comment:
@bulatgab Does this work okay? I would have thought you'd need to add an index file?

Reply:
Thanks for making that change! Approved.